pkg/providers/v1/aws.go

```diff
@@ -5089,6 +5089,13 @@ func IsFargateNode(nodeName string) bool {
 	return strings.HasPrefix(nodeName, fargateNodeNamePrefix)
 }
 
+// extract private ip address from node name
+func nodeNameToIpAddress(nodeName string) string {
```

Comment:
linter is complaining about the func name:
```
nodeNameToIpAddress should be nodeNameToIPAddress 

 golint -set_exit_status k8s.io/cloud-provider-aws/cmd/aws-cloud-controller-manager k8s.io/cloud-provider-aws/cmd/ecr-credential-provider k8s.io/cloud-provider-aws/pkg/apis/config k8s.io/cloud-provider-aws/pkg/apis/config/v1alpha1 k8s.io/cloud-provider-aws/pkg/controllers/options k8s.io/cloud-provider-aws/pkg/controllers/tagging k8s.io/cloud-provider-aws/pkg/providers/v1 k8s.io/cloud-provider-aws/pkg/providers/v2 k8s.io/cloud-provider-aws/pkg/providers/v2/mocks
/home/prow/go/src/k8s.io/cloud-provider-aws/pkg/providers/v1/aws.go:5093:6: func nodeNameToIpAddress should be nodeNameToIPAddress 
```
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/cloud-provider-aws/471/pull-cloud-provider-aws-check/1562936995934638080